test/jdk/java/nio/Buffer/GetChars.java

```diff
@@ -71,6 +71,19 @@ public void testSrcBeginIsNegative() {
                 () -> CB.getChars(-1, 3, new char[4], 0));
     }
 
+    @Test
+    public void testSrcBeginIsNegationOfPosition() {
```

Comment:
I skimmed through GetChars and I see it only tests a CB created with CharBuffer.wrap, only testGetChars tests all char buffers. All the test methods should be tested all char buffer implementations. So maybe we can create a follow-up issue to improve this test (and probably migrate it to a JUnit test too).

Reply:
Okay, let's create a follow-up issue for that.

Reply:
Could there also be value in a more generic `CharBufferAsCharSequenceTest` test which covers the entirety of the `CharSequence` interface on the various `CharBuffer` implementations in various "states" (position, offset, limit, etc.)?

Reply:
Please see https://bugs.openjdk.org/browse/JDK-8361715.